pypesto/objective/base.py

```diff
@@ -178,6 +183,8 @@ def __call__(
         x_full = self.pre_post_processor.preprocess(x=x)
 
         # compute result
+        if self.share_return_dict:
+            kwargs["return_dict"] = return_dict
```

Comment:
What if  `return_dict` is in kwargs already?

Reply:
It cannot be, since `return_dict` is it's own kwarg in this method's argument list.